.gitignore

```
@@ -1,2 +1,4 @@
 /dist/
 /src/readthedocs_cli.egg-info/
+__pycache__/
+.venv/
```

Would you anchor this to the repo root?

```suggestion
/.venv/
```